### ISSUE TITLE
ggpredict() will not work with type="sim"

### ISSUE THREAD

**Issue by rpmady** [strengejacke/ggeffects#60]
I tried to implement the alternative way to take in account model uncertainties by assigning type="sim" for my model object that I got using glmm.TMB(). But I receive an error.

```
p3 <- ggpredict(zip_mod5, terms="trmt", type="sim")
Error in match.arg(type) : 
  'arg' should be one of “fe”, “re”, “fe.zi”, “re.zi”, “surv”, “cumhaz”, “debug”
```

**Comment by strengejacke** [comment]
This also indicates that you're probably using an older version of ggeffects. `type = "sim"` is implemented since version 0.8.0. There are also tests for this option, so I bet it should definitely work.

**Comment by rpmady** [comment]
I updated R to 3.5.0 and we are in business. Sorry for bothering! Thank you for the guidance.
